client/landing/jetpack-cloud/sections/backups/restore/finished.tsx

```diff
@@ -1,19 +1,45 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
 
 interface Props {
-	restoreId: string;
-	siteId: number;
+	restoreTimestamp: string;
+	siteUrl: string | null;
 }
 
-const BackupRestoreFinished = ( { restoreId, siteId }: Props ) => (
-	<div>
-		<p>
-			Restore Finished for site { siteId } to { restoreId }
-		</p>
-	</div>
-);
+const BackupRestoreFinished: FunctionComponent< Props > = ( { restoreTimestamp, siteUrl } ) => {
+	const translate = useTranslate();
+
+	return (
+		<div>
+			<h3 className="restore__title">
+				{ translate( 'Your site has been successfully restored.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Your site has been successfully restored' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-cloud-restore-styling)